src/oemof/solph/flows/_investment_flow_block.py

```diff
@@ -887,6 +887,24 @@ def _objective_expression(self):
                         * lifetime
                         * ((1 + m.discount_rate) ** (-m.es.periods_years[p]))
                     )
+                    remaining_value = 0
+                    if lifetime > m.es.periods_matrix[p, -1]:
+                        remaining_lifetime = (
+                            lifetime - m.es.periods_matrix[p, -1]
+                        )
+                        remaining_annuity = economics.annuity(
+                            capex=m.flows[i, o].investment.ep_costs[-1],
+                            n=lifetime,
+                            wacc=interest,
+                        )
+                        remaining_value = (
+                            self.invest[i, o, p]
+                            * remaining_annuity
+                            * remaining_lifetime
+                        ) * (
+                            (1 + m.discount_rate) ** (-m.es.periods_years[-1])
```

Comment:
Why don't you use this formula, as you have just calculated the annuity? ![image](https://github.com/oemof/oemof-solph/assets/51374526/e46fb5ef-f1e4-4135-b775-49072599f7f9) [formula table](https://mathresearch.utsa.edu/wiki/index.php?title=Annuities#Formula_table)

Reply:
> Why don't you use this formula, as you have just calculated the annuity? ![image](https://user-images.githubusercontent.com/51374526/272875217-e46fb5ef-f1e4-4135-b775-49072599f7f9.png) [source](https://mathresearch.utsa.edu/wiki/index.php?title=Annuities#Formula_table)

You are totally right about that and I'm a tiny bit ashamed, I didn't implement it like this in the first place. Actually, this is what you need to transfer annuities, i.e. identical periodical payments, to NPVs. I'll also integrate this.

Reply:
The term 
$$\frac{(1 + i)^n -1}{i \cdot(1+i)^n}$$
is the reciprocal of the annuity factor. In German, it is called "Rentenbarwertfaktor". I think, there is no proper English equivalent for the term. I'm deriving it using `oemof.tools.economics.annuity()`